website/docs/r/timestreamwrite_database.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Timestream Write"
+layout: "aws"
+page_title: "AWS: aws_timestreamwrite_database"
+description: |-
+  Provides a Timestream database resource.
+---
+
+# Resource: aws_timestreamwrite_database
+
+Provides a Timestream database resource.
+
+## Example Usage
+
+### Basic usage
+
+```hcl
+resource "aws_timestreamwrite_database" "test_database" {
+  database_name = "database-example"
+}
+```
+
+### Full usage
+
+```hcl
+resource "aws_timestreamwrite_database" "test_database" {
+  database_name = "database-example"
+  kms_key_id    = aws_kms_key.foo.arn
+
+  tags = {
+    Name = "value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `database_name` – (Required) The name of the Timestream database. Minimum length of 3. Maximum length of 64.
+
+* `kms_key_id` - (Optional) The KMS key for the database. You can specify a key ID using any of the following:
```

Comment:
see kms comment above. some api take id in all formats but return Arn on describe. to prevents plan diffs if that's the case lets only keep the Arn format.